catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -519,7 +519,7 @@ public Response getDocument(@PathParam("sourceid") String sourceid, @PathParam("
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.INTERNAL_SERVER_ERROR);
             } catch (CatalogTransformerException e) {
-                String exceptionMessage = "Unable to transform Metacard.  Try different transformer: ";
+                String exceptionMessage = "Unable to transform Metacard : \n" + e.getMessage();
```

Comment:
Well, you need to be careful anytime you're referencing the exception in a message being sent back to an external client. You'd be much better off just sending back a canned message, like there was previously.